chirp/drivers/th_uvf8d.py

```diff
@@ -621,12 +621,6 @@ def _filter(name):
 
         return top
 
-        group.append(RadioSetting(
-                "disnm", "Display Name",
-                RadioSettingValueBoolean(_settings.disnm)))
-
-        return group
-
```

Comment:
Looks like this goes with the TODO above for implementing this, so I guess it's okay to just remove.